.github/drafts/16506/16506_number_slider_outline.md

```diff
@@ -0,0 +1,67 @@
+*** Range Slider ***
+
+The Number Slider widget allows users to choose a numerical value from a range of data.
+
+< TODO: NEED VIDEO DEMO >
+
+---
+# Usage
+- Example use case: Page turner/selector for a multi-page document or content
+
+---
+
+# Properties
+
+Properties allow you to edit the widget, connect it with other widgets and customize the user actions.
+
+## Widget Properties
+
+These properties allow you to edit the Number Slider widget. The following table lists all the widget's unique properties; if you do not see a certain widget property here, it may be listed under [Common Widget Properties]().
+
+| **Property** | **Description** |
+|-----------|-----------------|
+| **Min Value** | The starting/smallest possible value of the slider. The value may be negative and/or include decimals, however it must always be less than the **Max Value**. _(number)_ |
+| **Max Value** | The ending/largest possible value of the slider. The value may be negative and/or include decimals, however it must always be greater than the **Min Value**. _(number)_ |
+| **Step Size** | The increment by which the user can adjust the slider's value. This increment must be at least 0.1, and cannot be less than the **Min Value** or greater than the **Max Value**. _(number)_ |
```

Comment:
For the **Step Size** can we have a description similar to- `The amount by which the slider value should increase.`